filesystem/path.h

```diff
@@ -240,6 +241,15 @@ class path {
         return DeleteFileW(wstr().c_str()) != 0;
 #endif
     }
+    
+    bool create_directory() {
+#if defined(WIN32)
+        return SHCreateDirectory(nullptr, make_absolute().wstr().c_str()) == ERROR_SUCCESS;
```

Comment:
Any reason why you are using ``SHCreateDirectory`` (a deprecaded *shell* C function) rather than ``CreateDirectoryW``?